crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -372,6 +426,85 @@ pub fn prepare_core_3d_depth_textures(
     }
 }
 
+#[derive(Component)]
+pub struct ViewTransmissionTexture {
+    pub texture: Texture,
+    pub view: TextureView,
+    pub sampler: Sampler,
+}
+
+pub fn prepare_core_3d_transmission_textures(
+    mut commands: Commands,
+    mut texture_cache: ResMut<TextureCache>,
+    render_device: Res<RenderDevice>,
+    views_3d: Query<
+        (Entity, &ExtractedCamera, &Camera3d, &ExtractedView),
+        (
+            With<RenderPhase<Opaque3d>>,
+            With<RenderPhase<AlphaMask3d>>,
+            With<RenderPhase<Transmissive3d>>,
+            With<RenderPhase<Transparent3d>>,
+        ),
+    >,
+) {
+    let mut textures = HashMap::default();
+    for (entity, camera, camera_3d, view) in &views_3d {
+        let Some(physical_target_size) = camera.physical_target_size else {
+            continue;
+        };
+
+        // Don't prepare a transmission texture if the number of steps is set to 0
+        if camera_3d.transmissive_steps == 0 {
+            continue;
+        }
+
+        let cached_texture = textures
+            .entry(camera.target.clone())
+            .or_insert_with(|| {
+                let usage = TextureUsages::TEXTURE_BINDING | TextureUsages::COPY_DST;
+
+                // The size of the depth texture
+                let size = Extent3d {
+                    depth_or_array_layers: 1,
+                    width: physical_target_size.x,
+                    height: physical_target_size.y,
+                };
+
+                let format = if view.hdr {
+                    ViewTarget::TEXTURE_FORMAT_HDR
+                } else {
+                    TextureFormat::bevy_default()
+                };
+
+                let descriptor = TextureDescriptor {
+                    label: Some("view_transmission_texture"),
+                    size,
+                    mip_level_count: 1,
+                    sample_count: 1, // msaa.samples(),
```

Comment:
We need to handle MSAA somehow. Maybe disable the effect, but we can't have a crash.

Reply:
Not sure what you mean? Is it crashing with MSAA for you? It runs normally for me, just looks worse than TAA due to the lack of temporal jitter.

This `sample_count: 1` is just for the transmission texture, we don't actually render anything to it, just copy from the main texture to it, so AFAIK there's no need for a higher sample count.

I can just remove the count, or add an explanation about why it's set to `1`